sentry-android-core/src/test/java/io/sentry/android/core/SentryAndroidOptionsTest.kt

```diff
@@ -26,12 +26,12 @@ class SentryAndroidOptionsTest {
         assertEquals(BuildConfig.VERSION_NAME, sdkVersion.version)
 
         assertTrue(sdkVersion.packages!!.any {
-            it.name == "maven:sentry-android-core"
+            it.name == "maven:sentry-android-core" &&
```

Comment:
This was a funny bug. These tests were not testing the first condition at all but since Kotlin does allow this syntax the first like was just ignored.

On the side note - have you @bruno-garcia @marandaneto considered using AssertJ for assertions?

Reply:
I'd prefer we don't have than a single assert lib in usage, so if we were to change we'd need to replace everything. Too often new libraries creep in and we end up with a mix of stuff in the code base.

@marandaneto thoughts?

Reply:
I agree that mixing multiple assertions can lead to confusion. On the Java backend side, at least in what is considered a "modern Java backend", AssertJ has become a de-facto standard and it's been a while since I've used JUnit assertions.

Reply:
I wanted to use `assertk` which is an inspiration kotlin version of `AssertJ`, but we're trying to reuse/migrate the tests from the old sentry-java to this project, the idea was to gain some time, obviously, it didn't work because the code became quite different

Reply:
good catch @maciejwalkowiak , wondering If we've done the same mistake in other places :D